packages/app/src/sections/dashboard/Stake/EscrowTable.tsx

```diff
@@ -1,24 +1,36 @@
 import { ZERO_WEI } from '@kwenta/sdk/constants'
 import { EscrowData } from '@kwenta/sdk/types'
-import { truncateNumbers } from '@kwenta/sdk/utils'
+import { formatPercent, truncateNumbers } from '@kwenta/sdk/utils'
+import { wei } from '@synthetixio/wei'
 import { useCallback, useMemo, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { CellProps } from 'react-table'
 import styled from 'styled-components'
 
-import { DesktopOnlyView, MobileOrTabletView } from 'components/Media'
+import Badge from 'components/Badge'
+import Button from 'components/Button'
+import { Checkbox } from 'components/Checkbox'
+import { FlexDivCol, FlexDivRow, FlexDivRowCentered } from 'components/layout/flex'
+import { LargeScreenView, SmallScreenView } from 'components/Media'
 import Table from 'components/Table'
 import { TableCellHead, TableHeader } from 'components/Table'
+import { Body } from 'components/Text'
 import { StakingCard } from 'sections/dashboard/Stake/card'
 import { useAppDispatch, useAppSelector } from 'state/hooks'
-import { vestEscrowedRewards } from 'state/staking/actions'
+import { vestEscrowedRewardsV2 } from 'state/staking/actions'
+import { selectEscrowData, selectEscrowV2Data } from 'state/staking/selectors'
+import media from 'styles/media'
+import common from 'styles/theme/colors/common'
 
 import VestConfirmationModal from './VestConfirmationModal'
 
 const EscrowTable = () => {
 	const { t } = useTranslation()
 	const dispatch = useAppDispatch()
-	const escrowData = useAppSelector(({ staking }) => staking.escrowData)
+	const escrowV1Data = useAppSelector(selectEscrowData)
+	const escrowV2Data = useAppSelector(selectEscrowV2Data)
+	const escrowData = useMemo(() => [...escrowV1Data, ...escrowV2Data], [escrowV1Data, escrowV2Data])
```

Comment:
Could be good to move to a selector `combinedEscrowData`

Reply:
Yes. Updated. Thanks.